firmware/library/L0_LowLevel/startup.cpp

```diff
@@ -243,12 +243,27 @@ IsrPointer dynamic_isr_vector_table[] = {
     EepromIrqHandler,       // 56, 0xe0 - EEPROM
 };
 
-
 extern "C" void vPortSetupTimerInterrupt(void)  // NOLINT
 {
     // Empty implementation, startup handles this itself.
 }
 
+static StaticTask_t idle_task_tcb;
+static StackType_t idle_task_stack[configMINIMAL_STACK_SIZE];
+// Implementation of vApplicationGetIdleTaskMemory required when
+// configSUPPORT_STATIC_ALLOCATION == 1.
+// The function is called to statically create the idle task when
+// vTaskStartScheduler is invoked.
+extern "C" void vApplicationGetIdleTaskMemory(  // NOLINT
+    StaticTask_t ** ppx_idle_task_tcb_buffer,
+    StackType_t ** ppx_idle_task_stack_buffer,
+    uint32_t * pul_idle_task_stack_size)
+{
+    *ppx_idle_task_tcb_buffer   = &idle_task_tcb;
+    *ppx_idle_task_stack_buffer = idle_task_stack;
+    *pul_idle_task_stack_size   = configMINIMAL_STACK_SIZE;

```

Comment:
This will become a problem if the top array's stack size is changed without changing this assignment here. A better alternative is to use `SJ2_ARRAY_LENGTH(idle_task_stack)`. This will always follow the `idle_task_stack` regardless of what size it is, thus you never have to change multiple values for everything to work.